tools/cibadmin.c

```diff
@@ -677,19 +659,18 @@ main(int argc, char **argv)
         source = "STDIN";
         input = stdin2xml();
 
-#if ENABLE_ACL
     } else if (acl_eval_how != acl_eval_unused) {
-        username = uid2username(geteuid());
+        username = pcmk__uid2username(geteuid());
         if (pcmk_acl_required(username)) {
-            if (force_flag == TRUE) {
-                local_const_barf("The supplied command can provide skewed"
+            if (force_flag) {
+                fprintf(stderr, "The supplied command can provide skewed"
                                  " result since it is run under user that also"
                                  " gets guarded per ACLs on their own right."
                                  " Continuing since --force flag was"
                                  " provided.");
```

Comment:
The former macro added a newline automatically, you'll have to add it explicitly to the fprintf()'s